packages/client/hmi-client/src/components/code/tera-directory.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+	<aside class="min-w-10rem max-w-20rem">
+		<Tree
+			v-model:selectionKeys="selectedKey"
+			:value="directoryTree"
+			selectionMode="single"
+			:filter="true"
+			filterMode="lenient"
+			filter-placeholder="Find"
+			@node-select="onNodeSelect"
+		>
+		</Tree>
+	</aside>
+</template>
+
+<script setup lang="ts">
+import { onMounted, ref } from 'vue';
+import Tree, { TreeNode, TreeSelectionKeys } from 'primevue/tree';
+
+const directoryTree = ref<TreeNode[]>([]);
+
+const props = defineProps<{
+	files: string[];
+}>();
+
+const emit = defineEmits(['file-clicked']);
+
+const selectedKey = ref<TreeSelectionKeys>();
+onMounted(() => {
+	directoryTree.value = buildDirectoryTree(props.files);
+});
+
+function buildDirectoryTree(files) {
+	const root: TreeNode = {
```

Comment:
Could you make sure the root node is always open by default?
Even better if we can hide it, we do not need it on the interface.

Reply:
Done!

https://github.com/DARPA-ASKEM/terarium/assets/33158416/899786d3-7b4a-4ab8-a329-621dfffe4db2